### PR TITLE
fix countdown and allow to hide judging tab

### PR DESCRIPTION
**Pull request by casassg**
## What this PR does / Why we need it

- Fix deadline date
- Allow to hide judging tab

## Some questions
- [x] I have read the contributing guidelines
- [x] I abide by this repository Code of Conduct
- [x] I understand that my PR won't be merged until Travis gives a "green light"

### CODE REVIEW

File: app/hackathon_variables.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 HACKATHON_GITHUB_REPO = 'https://github.com/hackupc/registration/'
 
 # (OPTIONAL) Applications deadline
-HACKATHON_APP_DEADLINE = timezone.datetime(2019, 11, 1, 0, 0, tzinfo=timezone.pytz.timezone(TIME_ZONE))
+HACKATHON_APP_DEADLINE = timezone.datetime(2019, 10, 4, 23, 59, tzinfo=timezone.pytz.timezone(TIME_ZONE))
 # (OPTIONAL) When to arrive at the hackathon
 HACKATHON_ARRIVE = 'Registration opens at 3:00PM and closes at 6:00PM on Friday October 19th, ' \
```
Comment (MiguelSanchezP) [inline app/hackathon_variables.py]: Shouldn't the registration be "Friday October 11th"?

Comment (casassg) [reply app/hackathon_variables.py]: Registration closes a week before the event. That is Friday October 4th. 

Thanks for noticing though!

Comment (casassg) [reply app/hackathon_variables.py]: Oh, actually, you are right. I though you were talking about the deadline! Thanks for noticing! updating it now!

Comment (casassg) [reply app/hackathon_variables.py]: Update in #71  thanks for noticing!

Comment (MiguelSanchezP) [reply app/hackathon_variables.py]: @casassg Not a big deal, you're welcome!!

**Pull request merged**